test/setup.mjs

```diff
@@ -0,0 +1,96 @@
+// @note can't import shallow or ShallowWrapper specifically
+// import React from 'react'
+import Adapter from 'enzyme-adapter-react-16'
+import Enzyme from 'enzyme'
+// this is required because something overwrites the default Promise with core-js,
+// and the version of core-js sucks and breaks most of the tests.
+// So @babel/polyfill is used to fix it. Which it actually just uses a different version
+// of core-js to fix it.
+// @todo remove this bs when possible
+// import '@babel/polyfill'
```

Comment:
Looks like we can satisfy this todo and remove lines 5-10 now?

Reply:
yeah, this file was just copied from the shop project. I removed it